File: beginner-projects-solutions/99Bottles.c
#include <stdio.h>
int main ()
{
	printf("Lyrics of the Song 99 Bottles of Beer\n\n");
	int bottlecount=99;
	int i,s=1;
	int confirm=0;
	for (i=99;i>0;i--){
	printf("%d bottles of beer on the wall, %d bottles of beer. \n", bottlecount, bottlecount);
	bottlecount=bottlecount-s;
	printf("Take one down and pass it around, %d bottles of beer in the wall\n\n", bottlecount);
}
	printf("%d bottle of beer on the wall, %d bottle of beer. \n", bottlecount, bottlecount);
	bottlecount=bottlecount-s;
	printf("Take one down and pass it around, no more bottle of beer in the wall\n\n", bottlecount);
	if (confirm==bottlecount){
		printf("No more bottles of beer on the wall, no more bottle of beer. \n");
		printf("Go to store and buy some more, 99 bottles of beer.");
	}	
return 0;
}

File: beginner-projects-solutions/99Bottles.py
print("Lyrics of the Song 99 Bottles of Beer")
bottlecount = int(99)
confirm = int (1)
i = bottlecount
while(i>1):
    print(f"{bottlecount} bottles of beer on the wall, {bottlecount} bottles of beer. \n")
    bottlecount=bottlecount-1
    i-=1
    print(f"Take one down and pass it around, {bottlecount} bottles of beer in the wall \n\n")

print(f"{bottlecount} bottle of beer on the wall, {bottlecount} bottle of beer. \n")
print("Take one down and pass it around, no more bottle of beer in the wall \n\n")
print("No more bottles of beer on the wall, no more bottle of beer. \n")
print("Go to store and buy some more, 99 bottles of beer.")
